template-helpers/articles.js

```diff
@@ -164,6 +164,203 @@ const articleCopy = {
       },
     ],
   },
+  "steps-to-protect": {
+    paragraphs: [
+      {
+        lead: "Data breaches are one of many online threats. Using secure internet connections, updating your software, avoiding scam emails, and better password hygiene will help you stay safer while you browse.",
+      },
+      {
+        subhead: "Be wary of public wifi networks. ",
```

Comment:
I think "wifi" was the only other "typo" that **markdown-spellcheck** was flagging.
My guess is that this should be "Wi-Fi", per https://en.wikipedia.org/wiki/Wi-Fi and https://www.wi-fi.org/

```sh
npx markdown-spellcheck --en-us --ignore-acronyms --ignore-numbers --no-suggestions template-helpers/articles.js --report
```